money-monkey-web/app/ui/EditExpenseModal.tsx

```
@@ -0,0 +1,69 @@
+  initialExpense: Expense | null;
+  onClose: () => void;
+}
+
+export default function EditExpenseModal({
+  initialExpense,
+  onClose,
+}: EditExpenseModalProps) {
+  const handleSubmit = useCallback(
+    (expense: Expense, formikHelpers: FormikHelpers<Expense>) => {
+      console.log("New expense:", expense);
+      console.log("Helpers: " + formikHelpers);
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)